examples/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -274,6 +274,8 @@ service ABitOfEverythingService {
 				url: "https://github.com/grpc-ecosystem/grpc-gateway";
 				description: "Find out more about GetQuery";
 			}
+		  security: {
+      }
```

Comment:
The indendation of this line seems off, are you using spaces instead of tabs?

Reply:
Sorry, I've fixed it in https://github.com/grpc-ecosystem/grpc-gateway/pull/666/commits/a78ff380159daf7f0da4009c5b65a2263ad32249 (I'll squash this commit later)